RELEASING/send_email.py

```diff
@@ -61,7 +63,7 @@ def send_email(
         server.sendmail(sender_email, receiver_email, message)
 
 
-def render_template(template_file: str, **kwargs) -> str:
+def render_template(template_file: str, **kwargs: Any) -> str:
```

Comment:
can `kwargs` be `Dict[str, Any]`?

Reply:
`kwargs` are mostly represented as `Any` elsewhere in the code base. I think a follow up PR should make a sweep of the entire repo.